examples/with-nextjs-and-clerk-auth/src/monite-theme.d.ts

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+
+import '@mui/material/styles';
+import '@mui/x-data-grid/themeAugmentation';
+
+declare module '@mui/material/styles' {
+  interface TypeBackground {
+    default: string;
+    paper: string;
+    menu: string;
+    highlight: string;
+  }
+
+  interface Palette {
+    neutral: {
+      '10': string;
+      '30': string;
+      '50': string;
+      '70': string;
+      '80': string;
+      '90': string;
+      '95': string;
+      '100': string;
+    };
+  }
+
+  interface PaletteOptions {
+    neutral?: {
+      '10': string;
+      '30': string;
+      '50': string;
+      '70': string;
+      '80': string;
+      '90': string;
+      '95': string;
+      '100': string;
+    };
+  }
```

Comment:
Most of the types here are for changes that are needed in one or a couple places, and they all involve menus, which in my opinion is over-complication.